app/views/jobseekers/subscriptions/index.html.slim

```diff
@@ -18,14 +19,14 @@
 
           = f.govuk_submit t("jobs.sort_by.submit")
 
+  .govuk-grid-column-one-third.create-alert__button
+    = govuk_link_to(t(".button_create"), new_subscription_path(email: current_jobseeker.email, origin: jobseekers_subscriptions_path), button: true)
+
+.govuk-grid-row
   .govuk-grid-column-full
     .moj-filter-layout__content
       section
         - if @subscriptions.any?
-          .govuk-grid-row
-            .govuk-grid-column-full
-              = govuk_link_to(t(".button_create"), new_subscription_path(email: current_jobseeker.email, origin: jobseekers_subscriptions_path), button: true)
```

Comment:
i dont think this should only appear if you have existing subscriptions, should it?